xds/internal/testutils/balancer.go

```diff
@@ -20,27 +20,29 @@
 package testutils
 
 import (
-	"context"
 	"fmt"
-	"sync"
 	"testing"
 
-	corepb "github.com/envoyproxy/go-control-plane/envoy/api/v2/core"
-	"google.golang.org/grpc"
 	"google.golang.org/grpc/balancer"
 	"google.golang.org/grpc/connectivity"
-	"google.golang.org/grpc/internal/wrr"
 	"google.golang.org/grpc/resolver"
-	"google.golang.org/grpc/xds/internal"
 )
 
-const testSubConnsCount = 16
+// DefaultTestSubConnsCount is the number of TestSubConns initialized as part of
+// package init.
+const DefaultTestSubConnsCount = 16
```

Comment:
Why did you export this?

And since this value is read during `init()`, I don't think there's any way to set it.

Reply:
The main reason for exporting is to make it obvious how many subConns are being initialized. My motivation was not to make it settable by users of this package. If we need to be able to do that, we can make this a var at a later point in time. But the main reasoning being that this is a package which exports some stuff, and since it initializes a bunch of subConns, we should make it obvious how many are initialized.

Reply:
SG.
One nit is to remove `Default` from the name.